3rdParty/Radon/Radon/source/File.cpp

```diff
@@ -47,15 +47,15 @@ namespace radon
 
 	Section *File::getSection(const std::string & name)
 	{
-		for (auto & section : sections)
+		for (int i = 0; i < (int)sections.size(); i++)
```

Comment:
We cannot use iterators? Can you elaborate on this?

Reply:
IIRC this was done for the attempt at an XBox port. XBox SDK at that point only supported C++98

Reply:
(I'm not the author of these changes)

Reply:
C++03